interface/weno/transmitrx.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * File use to transmit Rx
+ *
+ * @package OpenEMR
+ * @link    http://www.open-emr.org
+ * @author  Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2019 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+require_once "../globals.php";
+
+use OpenEMR\Rx\Weno\NewRx;
+
+if (isset($_GET['scripts'])) {
+    $list = filter_input(INPUT_GET, 'scripts', FILTER_VALIDATE_INT);
+} else {
+    echo xlt("Error: No prescriptions sent");
+    exit;
+}
+
+$meds = explode(",", $list);
```

Comment:
Bit confused on line above, since it looks like code above is only letting $list be equal to a integer?
So, appears this script only does one med. If so, then don't need line above or the foreach below.

Reply:
the list is a list of numbers. So, I am not setting the value to an INT. It is 123,345,567,987 string. So, I thought the validate INT would be correct. 
The string that is being sent over can be one med or more. It is sent a comma-separated list.